client/driver/rkt.go

```diff
@@ -53,20 +55,27 @@ type RktDriverConfig struct {
 
 // rktHandle is returned from Start/Open as a handle to the PID
 type rktHandle struct {
-	proc        *os.Process
-	image       string
-	logger      *log.Logger
-	killTimeout time.Duration
-	waitCh      chan *cstructs.WaitResult
-	doneCh      chan struct{}
+	pluginClient *plugin.Client
+	userPid      int
+	executor     executor.Executor
+	taskDir      string
```

Comment:
You are never setting the taskDir in the handle. Also the h.taskDir is not referenced anywhere else.